roles/flannel/meta/main.yml

```diff
@@ -14,3 +14,4 @@ galaxy_info:
   - system
 dependencies:
 - { role: openshift_facts }
+- { role: etcd_client_certificates, etcd_generated_certs_dir: /etc/etcd/generated_certs, etcd_cert_prefix: node.etcd- }
```

Comment:
That said, we should also probably make the destination directory for the certificates configurable.